src/sass/components/govcy-text-inputs.scss

```diff
@@ -0,0 +1,132 @@
+  $border-radius : 1px;
+  $govcy-outline-color:yellow;
+  
+  
+    textarea:focus,
+    input[type="text"]:focus,
+    input[type="password"].focus,
+    input[type="datetime"]:focus,
+    input[type="datetime-local"]:focus,
+    input[type="date"]:focus,
+    input[type="month"]:focus,
+    input[type="time"]:focus,
+    input[type="week"]:focus,
+    input[type="number"]:focus,
+    input[type="email"]:focus,
+    input[type="url"]:focus,
+    input[type="search"]:focus,
+    input[type="tel"]:focus,
+    input[type="color"]:focus,
+    .uneditable-input:focus {   
+       border-color: gray; 
```

Comment:
I think this should be black maybe?

The figma prototypes have a black border
https://www.figma.com/file/zjXeMiVsDDt1QO8te0hoob/DMRID-%E2%80%93-Components?node-id=54%3A65

Reply:
![Screenshot 2021-12-13 at 10 14 05](https://user-images.githubusercontent.com/3469840/145793773-203dddf1-1b6b-429b-b103-c4af0fcf855b.png)
<img width="1779" alt="Screenshot 2021-12-13 at 10 14 14" src="https://user-images.githubusercontent.com/3469840/145793776-7fe1cf4f-6d5c-403b-b72c-cd8892db8629.png">